### PR TITLE
Add docs for various `invalid-*` messages

### PR DESCRIPTION
**Pull request by clavedeluna**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Add docs examples for various `invalid-*` messages

Refs #5953

### CODE REVIEW

File: doc/data/messages/i/invalid-class-object/bad.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class A:
+    pass
+
+
+A.__class__ = 1  # [invalid-class-object]
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-class-object/bad.py:5]:
```suggestion
class Apple:
    pass


Apple.__class__ = 1  # [invalid-class-object]
```

File: doc/data/messages/i/invalid-class-object/good.py [hunk 2]
```diff
@@ -1 +1,9 @@
-# This is a placeholder for correct code for this message.
+class A:
+    pass
+
+
+class B:
+    pass
+
+
+A.__class__ = B
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/i/invalid-class-object/good.py:9]:
```suggestion
class Apple:
    pass


class RedDelicious:
    pass


Apple.__class__ = RedDelicious
```

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Thank you !  I'm jst trying to avoid Foo/Bar/FooBar and boring one letter name. I often try to have fruit and animals in example so it's more "lively". Other example could work too, let me know if it make no sense. Admittedly I did not find the use case for changing ``__class__`` live so it might be off..

**Comment by clavedeluna** [comment]
Github's batch commit feature must be down temporarily 🤔

**Review by Pierre-Sassoulas — APPROVED** [review]
👌

**Pull request merged**